lib/model/folder_sendrecv.go

```diff
@@ -1536,7 +1536,7 @@ loop:
 	for {
 		select {
 		case <-f.ctx.Done():
-			state.fail(errors.Wrap(f.ctx.Err(), "folder stopped"))
+			state.fail(fmt.Errorf("folder stopped: %w", f.ctx.Err()))
```

Comment:
Are we sure that `Err()` will be non-nil?  Comment might help in that case.

Reply:
We are; context docs say "// If Done is closed, Err returns a non-nil error explaining why:". I don't think really requires explanation, it wouldn't be a question on anyone's mind unless the old Wrap() function had the odd-ish feature of wrapping nil as nil.